src/main/java/core/basesyntax/service/impl/StorageServiceImpl.java

```diff
@@ -0,0 +1,32 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.service.StorageService;
+import java.util.Map;
+
+public class StorageServiceImpl implements StorageService {
+    private final Storage storage = new Storage();
+
+    public StorageServiceImpl() {
+    }
```

Comment:
redundant constructor we have it available by default if we do not have any others